src/migrators/schema.ts

```diff
@@ -0,0 +1,18 @@
+export const data = {
+    migrations: [
+        {
+            "satellite_body": [
+                "-- The ops log table\nCREATE TABLE IF NOT EXISTS _electric_oplog (\n  rowid INTEGER PRIMARY KEY AUTOINCREMENT,\n  namespace String NOT NULL,\n  tablename String NOT NULL,\n  optype String NOT NULL,\n  primaryKey String NOT NULL,\n  newRow String,\n  oldRow String,\n  timestamp TEXT\n);",
+                "-- Somewhere to keep our metadata\nCREATE TABLE IF NOT EXISTS _electric_meta (\n  key TEXT,\n  value BLOB\n);",
+                "-- Somewhere to track migrations\nCREATE TABLE IF NOT EXISTS _electric_migrations (\n  id INTEGER PRIMARY KEY AUTOINCREMENT,\n  name TEXT NOT NULL UNIQUE,\n  sha256 TEXT NOT NULL,\n  applied_at TEXT NOT NULL\n);",
+                "-- Initialisation of the metadata table\nINSERT INTO _electric_meta (key, value) VALUES ('compensations', 0), ('lastAckdRowId','0'), ('lastSentRowId', '0'), ('lsn', 'MA=='), ('clientId', '');",
+                "-- These are toggles for turning the triggers on and off\nDROP TABLE IF EXISTS _electric_trigger_settings;",
+                "CREATE TABLE _electric_trigger_settings(tablename STRING PRIMARY KEY, flag INTEGER);"
+            ],
+            "encoding": "escaped",
+            "name": "1666288242_init",
```

Comment:
Question: `electric migrations init` command as of right now also creates an init migration. Is that going to be a problem that this kinda hijacks the whole "migrations are outside the app" thing? I would have thought that we were just going to run `CREATE TABLE IF NOT EXISTS` on every electrify call just in case, or is that unreasonable and I'm missing something crucial?

I'm aware that the current body also does drop tables and insert rows, but we can do that conditionally at startup as well

Reply:
> but we can do that conditionally at startup as well

I agree that such approach would be preferred. Better not to mix data-related schema migration with application-related migration

Reply:
I agree that all the discussion points are relevant. I just did enough to make the code work again after the breaking change on CLI.

The migrations up() is called down the execution of electrify. I am currently re-using the migrator code that is there, we will need to adapt that if Satellite stops applying any migrations, but I think that can be handled separately.

The same with conditional and separating data from schema, but I'd handle that in separate tickets, with more time.

Having said that, I am open to any strong requirements that any of you think needs to be addressed rightaway.

Reply:
I feel like that's a little too M in MVP for my liking, but I'm ok as long as we fix this after general flow works at least once :D

I'm going to mildly wave a flag of "hey we wanted to do chained migrations shas @paulharter and this may affect this" but that's a story for another time anyway :)